packages/ui/src/components/shared/settings/__test__/SettingsWalletButton.spec.tsx

```diff
@@ -49,34 +62,39 @@ describe('SettingsWalletButton', () => {
       });
     });
 
-    it('should switch to BackupSecretPhraseDialog content when clicking on `Backup secret recovery phrase` button', async () => {
+    it('should switch to BackupSecretPhraseDialog content when clicking on `Backup Secret Recovery Phrase` button', async () => {
       const BackupSecretRecoveryPhraseButton = await screen.findByRole('button', {
-        name: /Backup secret recovery phrase/,
+        name: /Backup Secret Recovery Phrase/,
       });
 
-      user.click(BackupSecretRecoveryPhraseButton);
+      await user.click(BackupSecretRecoveryPhraseButton);
 
       await waitFor(() => {
-        expect(screen.getByText(/Backup secret recovery phrase/)).toBeInTheDocument();
+        expect(screen.getByText(/Backup Secret Recovery Phrase/)).toBeInTheDocument();
         expect(screen.getByText(/reveal the secret recovery phrase/)).toBeInTheDocument();
         expect(screen.getByRole('button', { name: /View Secret Recovery Phrase/ })).toBeInTheDocument();
         expect(screen.getByRole('button', { name: /Back/ })).toBeInTheDocument();
       });
     });
 
-    it('should show ThemeModeButton, LanguageSelection, AutoLockSelection, `Backup secret recovery phrase` button, `Change wallet password` button', async () => {
-      expect(await screen.findByRole('dialog')).toBeInTheDocument();
+    it('should switch to ChangeWalletPasswordDialog content when clicking on `Change Wallet Password` button', async () => {
+      const ChangeWalletPasswordButton = await screen.findByRole('button', { name: /Change Wallet Password/ });
+
+      await user.click(ChangeWalletPasswordButton);
 
       await waitFor(() => {
-        expect(screen.getByRole('button', { name: /Dark/ })).toBeInTheDocument();
-        expect(screen.getByRole('button', { name: /Light/ })).toBeInTheDocument();
-        expect(screen.getByRole('button', { name: /System/ })).toBeInTheDocument();
-        expect(screen.getByRole('button', { name: /Close settings/ })).toBeInTheDocument();
-        expect(screen.getByRole('button', { name: /English/ })).toBeInTheDocument();
-        expect(screen.getByRole('button', { name: /5 minutes/ })).toBeInTheDocument();
-        expect(screen.getByRole('button', { name: /Backup secret recovery phrase/ })).toBeInTheDocument();
+        expect(screen.getByText(/Change Wallet Password/)).toBeInTheDocument();
+        expect(screen.getByText(/Enter your wallet password to continue/)).toBeInTheDocument();
+        expect(screen.getByRole('button', { name: /Continue/ })).toBeInTheDocument();
+        expect(screen.getByRole('button', { name: /Back/ })).toBeInTheDocument();
       });
     });
+
+    it('should show ThemeModeButton, LanguageSelection, AutoLockSelection, `Backup Secret Recovery Phrase` button, `Change Wallet Password` button', async () => {
```

Comment:
For short, we could do:
```suggestion
    it('should show SettingsWalletDialog', async () => {
```
instead of listing all of the buttons, components in the view.